src/CommunityToolkit.Maui.SourceGenerators/Helpers/IsExternalInit.cs

```diff
@@ -0,0 +1,66 @@
+﻿// <auto-generated>
+//   This code file has automatically been added by the "IsExternalInit" NuGet package (https://www.nuget.org/packages/IsExternalInit).
+//   Please see https://github.com/manuelroemer/IsExternalInit for more information.
+//
+//   IMPORTANT:
+//   DO NOT DELETE THIS FILE if you are using a "packages.config" file to manage your NuGet references.
```

Comment:
Can we remove this file and add it to gitignore?

Reply:
No, this file is needed in order to use `records` in `netstandard`

Reply:
But it is autogenerated by the package: IsExternalInit. However I do not see where it is used

Reply:
But we don't use the package, we just have this file on our own. Without this file, we can't compile the SG when using `record`, so we don't use this directly. This explanation was added to the summary of the method.
You can find more info [here](https://stackoverflow.com/questions/67687078/can-we-use-records-in-c-sharp-8-0)

Reply:
Yup, this is a known-issue in .NET Standard 2.0. 

I had to add this class to all of my Xamarin.Forms apps where I used `record` until .NET Standard 2.1 came out.